Tests/CartonCommandTests/InitCommandTests.swift

```diff
@@ -73,4 +73,84 @@ final class InitCommandTests: XCTestCase {
     // finally, clean up
     try packageDirectory.delete()
   }
+
+  func testInitWithTokamakTemplate() throws {
+    // given I've created a directory
+    let package = "fusion"
+    let packageDirectory = testFixturesDirectory.appending(component: package)
+    let expectedTemplateSource =
```

Comment:
Could this string be moved outside of the function body? Then the linter warning about the function body line count would go away.

Reply:
It can. Easy fix.